src/test/java/com/faforever/client/game/GameServiceTest.java

```diff
@@ -133,9 +137,7 @@
   public void setUp() throws Exception {
     junitPlayer = PlayerBuilder.create("JUnit").defaultValues().get();
 
-    ClientProperties clientProperties = new ClientProperties();
-
-    instance = new GameService(clientProperties, fafService, forgedAllianceService, mapService,
+    instance = new GameService(new ClientProperties(), fafService, forgedAllianceService, mapService,
```

Comment:
No need for changing this why do u change it?

Reply:
I can no longer remember. I had to do this as a result of a change I made in production code. But as it seems this is no longer needed (obviously because one of the reverts I had to do). I changed it back to the original implementation before my changes.